docs/en/setup/overview.md

```diff
@@ -15,7 +15,8 @@ The SkyWalking Rover requires specialized protocols to communicate with SkyWalki
 
 | SkyWalking Rover Version | SkyWalking OAP |
 |--------------------------|----------------|
-|0.1.0+                    | \> = 9.1.0     |
+| 0.1.0+                   | \> = 9.1.0     |
```

Comment:
Let's remove 0.1, after all, we changed the landscape of the project in 0.6. We should make this changing very clear in the release note.